omf/scripts/omf_to_geoh5.py

```diff
@@ -0,0 +1,19 @@
+import sys
+from pathlib import Path
+
+from omf.fileio import OMFReader
+from omf.fileio.geoh5 import GeoH5Writer
+
+
+def run():
+    omf_filepath = Path(sys.argv[1])
+    if len(sys.argv) < 2:
+        output_filepath = omf_filepath.with_suffix(".geoh5")
+    else:
+        output_filepath = Path(sys.argv[2])
+    reader = OMFReader(omf_filepath.absolute())
```

Comment:
Similar issue here with `Path` instead of `str`

AttributeError: 'WindowsPath' object has no attribute 'seek'
Exception ignored in: <function OMFReader.__del__ at 0x000001E59D1FA160>
Traceback (most recent call last):
  File "C:\Users\dominiquef\Documents\GIT\mira\omf\omf\fileio\fileio.py", line 110, in __del__
    self._fopen.close()
AttributeError: 'WindowsPath' object has no attribute 'close'